repos.d/centos7-cloudwatch-agent.repo

```diff
@@ -0,0 +1,12 @@
+[cloudwatch-agent-el7]
```

Comment:
what is this file used by?
cosmos deploy uses https://github.com/bbc/digital-paper-edit-infrastructure/blob/cosmos-infra/infrastructure/repositories.json, and I don't see cloudwatch-agent being required in any specfile.